libs/designsystem/button/src/button.component.ts

```diff
@@ -143,6 +143,11 @@ export class ButtonComponent implements AfterContentInit {
       return;
     }
 
+    // Ensure buttons inside the toolbar is always shown as icon only:
```

Comment:
This comment is not quite true if the button is text only, right?

Reply:
Good catch - fixed 👍🏻